components/linking_landing_page/linking_landing_page.tsx

```diff
@@ -388,10 +390,12 @@ export default class LinkingLandingPage extends PureComponent<Props, State> {
                     >
                         {this.renderCheckboxIcon()}
                     </button>
-                    <FormattedMessage
-                        id='get_app.rememberMyPreference'
-                        defaultMessage='Remember my preference'
-                    />
+                    <span
+                        onClick={this.handleChecked}
+                        className={'get-app__checkbox--label'}
+                    >
+                        {formatMessage({id: 'get_app.rememberMyPreference', defaultMessage: 'Remember my preference'})}
+                    </span>
```

Comment:
Could we bring the span inside of button and play around styling to solve this issue instead?